en/guide/device_version.md

```diff
@@ -6,24 +6,35 @@ The [Info](../api_reference/classdronecore_1_1_info.md) class is used to get dev
 hardware may return garbage values (for example, the simulator provides garbage values for the vendor 
 firmware semantic version).
 
-The code below shows how to query the UUID, version, and product, information and print it to the console:
+## Preconditions
 
+The following code assumes that you already have included DroneCore (`#include <dronecore/dronecore.h>`) and that there is a [connection to a device](../guide/connections.md) obtained as shown below:

```

Comment:
@julianoes Conscious decision here NOT to include mention of also having to link to the plugin library. Do you think this has to be mentioned? We do link to the using plugins doc that covers this sort of thing.

Reply:
You now need to include and link to each plugin that you require. So in this case we do require info.

Reply:
I had hoped that having that in the "Using Plugins" section would be sufficient (since all the sections otherwise have to duplicate similar instructions). Anyway, I've updated so that this is in every docs as "Create the Plugin" (rather than preconditions). OK?